### DISCUSSION TITLE
Allow filtering out certain tags by default, like with users

### DISCUSSION THREAD

**Discussion by lendrick** [civitai/civitai#131] [Ideas]
Everybody probably has a few things on here that they're just not into and would rather not see models for.  It would be nice to be able to permanently filter out models by tag.

**Comment by JustMaier** [comment]
**This is live!**

You can choose to hide any tag associated with a model:
![image](https://user-images.githubusercontent.com/607609/212382856-645741be-fb8f-43fd-bb33-55b4ccf8ef50.png)
![image](https://user-images.githubusercontent.com/607609/212386364-6e532dcb-e942-4b31-8463-44acf227de68.png)

And if you ever want to review or adjust the tags that you've hidden you can do so from your account settings:
![image](https://user-images.githubusercontent.com/607609/212386607-6cc126b7-3fc3-4468-9976-5f48768865e3.png)

**Reply by jansteffen** [reply]
Where? I'm not seeing this option... 
![image](https://user-images.githubusercontent.com/1278331/224024288-abde685a-7f1b-4229-9c4f-3cf75fe672d5.png)
